### PR TITLE
Create federation token for user prefixes on S3

### PR DESCRIPTION
**Pull request by audax**
With a post request to the `token` resource of the REST api, a user can get a [federation token](https://docs.aws.amazon.com/STS/latest/APIReference/API_GetFederationToken.html) with specific permission which allow him to list and modify all objects in the qabel bucket with a prefix specific to this user (currently the user id from accounting).

This code depends on having AWS credentials available and configured as described in the patched readme.

Depends on #8

### CODE REVIEW

**Comment by thechauffeur** [comment]
Please reference the PR which must be merged prior to this one.

**Comment by audax** [comment]
Depends on #8

File: qabel_provider/aws.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    def __init__(self, user):
+        self.user = user
+        self.policy = {
+            "Version": "2012-10-17",
+            "Statement": []
+        }
+        self.append(
+            {
+                "Sid": "Stmt1444045543000",
```
Comment (jan-schreib) [inline qabel_provider/aws.py]: Out of curiosity, what is this?

Comment (audax) [inline qabel_provider/aws.py]: This defines the policy version and sets a statement id which is completely irrelevant in this case. The Sid is optional and can be removed, which is what I will do.

Comment (jan-schreib) [inline qabel_provider/aws.py]: You will remove the Sid when?

Comment (audax) [inline qabel_provider/aws.py]: Yup.

**Comment by jan-schreib** [comment]
I don't see any issues.

**Pull request merged**